apps/console/src/features/console-settings/components/console-roles/create-console-role-wizard/create-console-role-wizard-basic-info-form.tsx

```diff
@@ -63,11 +53,13 @@ const CreateConsoleRoleWizardBasicInfoForm: FunctionComponent<CreateConsoleRoleW
             helperText={ (
                 <Hint>
                     Provide a distinctive and meaningful display name for the role. This name should clearly represent
-                    the purpose or responsibilities associated with this role within the Console application. E.g.{ " " }
-                    <Code>application:read</Code>
+                    the purpose or responsibilities associated with this role within the Console application.
+                    E.g.{ " " }<Code>application:read</Code>
```

Comment:
Ideally this should have localization.

Reply:
Yes, but this was not added from this PR. This should be fixed in a separate effort [1].

[1] https://github.com/wso2/identity-apps/pull/4747#issuecomment-1820144558

Reply:
Ack